src/htmx.js

```diff
@@ -2171,8 +2171,8 @@ var htmx = (function() {
           if (eventFilter) {
             triggerSpec.eventFilter = eventFilter
           }
+          consumeUntil(tokens, NOT_WHITESPACE)
           while (tokens.length > 0 && tokens[0] !== ',') {
-            consumeUntil(tokens, NOT_WHITESPACE)
```

Comment:
I feel like this consumeUntil should remain in the loop so we support white space inside the comma separated values, no?